### PR TITLE
Adding a section for OpenSearch SQL CLI on Downloads/Getting Started page to fix #654

### LINKED ISSUE #654: Add a section for OpenSearch SQL CLI on Downloads/Getting Started page.
Add a section for OpenSearch SQL CLI on [Downloads/Getting Started page](https://opensearch.org/downloads.html#opensearch). It could be 
- brief descriptive words for this cli tool
- links to Pypi https://pypi.org/project/opensearchsql/
- command to install `pip install opensearchsql`

### PR DESCRIPTION
**Pull request by vimalMK**
Signed-off-by: Vimal K <vimalinfo10@gmail.com>

### Description
Add a section for OpenSearch SQL CLI on Downloads/Getting Started page
 
The new section looks as follows:
193970841-a6239f63-1178-4e6f-9833-3e68bc385672

The brief descriptive words for this cli tool was taken from https://pypi.org/project/opensearchsql/ . I did not had that information so I took the liberty to add what I feel fits well for the section. The platform and package information is also added, open to change those if required. This is my very first contribution to OpenSearch project , so please let me know if anything has to be changed . Thank you

### Issues Resolved
Closes #654 

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by zhongnansu — APPROVED** [review]
lgtm

**Comment by zhongnansu** [comment]
@opensearch-project/project-website Can we have maintainers to take a look and PR, and trigger the CI?

File: _includes/downloads/opensearch-sql-cli.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<p>The SQL CLI component in OpenSearch is a stand-alone Python application and can be launched by a 'wake' word <code>opensearchsql</code>.</p>
+<p>It only supports <a href="https://docs-beta.opensearch.org/search-plugins/sql/index/" rel="nofollow">OpenSearch SQL Plugin</a>
+    You must have the OpenSearch SQL plugin installed to your OpenSearch instance to connect.
```
Comment (wbeckler) [inline _includes/downloads/opensearch-sql-cli.html:3]: I would suggest "installed in your . . ."

**Comment by vimalMK** [comment]
@wbeckler Thank you for the review. Will patch those changes soon. :)

**Comment by vimalMK** [comment]
PR patched!

**Comment by zhongnansu** [comment]
@wbeckler could you take another look, thx

**Review by wbeckler — APPROVED** [review]

**Comment by Yury-Fridlyand** [comment]
Upvoting this.
Looking forward to have this published!

**Comment by nateynateynate** [comment]
@vimalMK - I apologize for letting this sit for so long. No reason we shouldn't have a download artifact for it. I have a few questions. 

Will these artifacts need to be updated and recommitted every time theres a new CLI version? I think this might be something we'd want to tie in with your release process so that we can make sure the artifacts are updated. 

How often do these get updated?

**Comment by zhongnansu** [comment]
> @vimalMK - I apologize for letting this sit for so long. No reason we shouldn't have a download artifact for it. I have a few questions.
> 
> Will these artifacts need to be updated and recommitted every time theres a new CLI version? I think this might be something we'd want to tie in with your release process so that we can make sure the artifacts are updated.
> 
> How often do these get updated?

@nateynateynate SQL-CLI release has its own versioning, doesn't have to go with every OpenSearch release, and we don't expect it to be released very often. So far it has only been released once. https://pypi.org/project/opensearchsql/#history

**Comment by dblock** [comment]
@nateynateynate What would you like to do with this?

**Comment by nateynateynate** [comment]
When #3112 publishes I'll close this one out. Thanks everyone.

**Comment by nateynateynate** [comment]
Looks like you're live! #3117

**Pull request closed without merging**